config/config.exs

```diff
@@ -1,4 +1,2 @@
 use Mix.Config
 
-config :bugsnag,
```

Comment:
Ah. Rereading I see it now. It'd be nice to separate those changes, just so their intentions are clear. I might pull the config change into a separate commit which you could rebase these commits on top of to get a clean and clear history. How does that sound?